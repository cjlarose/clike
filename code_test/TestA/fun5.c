


// 8 args

void printint(int);
void printchar(char);

int f(x, y, z, a,b,c,d,e)
   int x, y, z;
   char a,b,c,d,e;
{
  return  (e)*(a + b * (c - d) + x * y + z);
}


void main()
{
  int x, y;

  x = 17;
  y = f(x, x*2, x/3, 1,2,3,4, 5);

  printint(y);
}

