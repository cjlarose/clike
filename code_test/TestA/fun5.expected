Loaded: /usr/local/share/spim/exceptions.s
2910